src/ui/win32/bindings/MemoryViewerControlBinding.cpp

```diff
@@ -236,6 +238,30 @@ bool MemoryViewerControlBinding::HandleNavigation(UINT nChar)
     }
 }
 
+bool MemoryViewerControlBinding::HandleShortcut(UINT nChar)
+{
+    const bool bShiftHeld = (GetKeyState(VK_SHIFT) < 0);
+    const bool bControlHeld = (GetKeyState(VK_CONTROL) < 0);
+    auto nModifier = 1;
+
+    // Increase by 1 or 2 on the high and lower nibble depending key pressed
+    if (bControlHeld)
+        nModifier *= 2;
+    if (bShiftHeld)
+        nModifier *= 16;
```

Comment:
These `nModifier` calculations should be in the case statements. You shouldn't be doing the calculations for things like copy/paste.